WhiteDentalClinic/WhiteDentalClinic.Application/Services/AppointmentService.cs

```diff
@@ -0,0 +1,103 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using WhiteDentalClinic.Application.Models.AppointmentModel;
+using WhiteDentalClinic.Application.Models.Dentist;
+using WhiteDentalClinic.Application.Services.Interfaces;
+using WhiteDentalClinic.DataAccess.Entities.AppointmentEntity;
+using WhiteDentalClinic.DataAccess.Entities.DentistEntity;
+using WhiteDentalClinic.DataAccess.Repositories.AppointmentRepository;
+
+namespace WhiteDentalClinic.Application.Services
+{
+    public class AppointmentService : IAppointmentService
+    {
+        private readonly IAppointmentRepository _appointmentRepository;
+        private readonly IMapper _mapper;
+        public AppointmentService(IAppointmentRepository appointmentRepository, IMapper mapper)
+        {
+            _appointmentRepository= appointmentRepository;
+            _mapper= mapper;
+        }
+        public IEnumerable<AppointmentResponseModel> GetAllAppointments()
```

Comment:
One common approach is not to use "appointment" in all method naming considering that you are already in an "AppointmentService". What you did is not wrong but could be simplified. Basically whenever you call these methods you can get from` appointmentService.GetAllAppointments()` to `appointmentService. GetAll()` and you kind of know that you get all the appointments considering the service you are calling is an appointments one which should have as main responsibility handling appointments. (applicable as a general rule)